### DISCUSSION TITLE
HLS live sream has lip sync issue

### DISCUSSION THREAD

**Discussion by ktvnet** [ant-media/Ant-Media-Server#5823] [FAQs]
I have a new Ant Media server community edition 2.7.0. with an RTMP ingest and HLS playback. The HLS playback has a very bad lip sync issue. How do I correct this?

**Comment by tremzo** [comment]
@ktvnet Do you mean the audio/video is not syncing very well? If that's the case, please can you make sure you are live streaming according to our recommended settings here https://antmedia.io/docs/guides/publish-live-stream/rtmp/publish-with-obs/#tune-for-ultra-low-latency-streaming and let us know if that solves the issue.
